charts/operator-wandb/charts/weave-trace/values.yaml

```diff
@@ -6,6 +6,13 @@ image:
   tag: latest
   pullPolicy: Always
 
+bufstream:
+  host: bufstream.bufstream.svc.cluster.local
+  port: 9092
+
+weaveWorker:
+  enabled: true
```

Comment:
This needs to be defaulted to false, and the test configs should be updated to test with and without this enabled